content/foundations/color/base-scales.mdx

```diff
@@ -3,6 +3,71 @@ title: Base scales
 description:
   Primer base color scales makeup the foundation for all UI color considerations.
 ---
+## Neutral color scales
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labeled from 0 to 11. The top scale is the light scale, which starts with white and ends with black. The bottom scale is the dark scale, which starts with black and ends with white."
+  src="https://github.com/user-attachments/assets/3081e29d-6581-401b-91da-c014eee9cf66"
+/>
+
+Primer's neutral scales offer white, black, and shades of gray between 0 and 11. There are two versions of the neutral scale: light, and dark. The light and dark scales are inverted respectively, with the light scale starting with white and the dark scale starting with black. By inverting the scales, light and dark themes are able to share many of the same functional color tokens without custom overrides.
```

Comment:
```suggestion
Primer's neutral scales offer white, black, and shades of gray between 0 and 11. There are two versions of the neutral scale: light, and dark. 

The light and dark scales are inverted respectively, with the light scale starting with white and the dark scale starting with black. By inverting the scales, light and dark themes are able to share many of the same functional color tokens without custom overrides.
```

This is a small nit, but I'm thinking we separate this into two separate paragraphs to both aid in reading ease, but also reinforce that A) there are two versions of the neutral scale, and then B) the two scales allow for increased color value sharing.